exporters/variables-scss/src/generators/cssObjectGenerator.ts

```diff
@@ -1,51 +1,56 @@
-import { Token, TokenGroup } from '@supernovaio/sdk-exporters';
-import { toPlural } from '../helpers/stringHelper';
+import { Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
 import { tokenVariableName } from '../helpers/tokenHelper';
+import { toPlural } from '../helpers/stringHelper';
 
-export const generateObjectContent = (
-  tokens: Array<Token>,
-  tokenGroups: Array<TokenGroup>,
-  withParent: boolean,
-): string => {
-  return tokens.reduce((result, token) => {
-    const name = tokenVariableName(token, tokenGroups, withParent);
-    const numericPart = name.match(/\d+/)?.[0];
-    const prefix = `${token.origin?.name?.split('/')[0].toLowerCase()}-`;
-    const nonNumericPart = name.replace(prefix, '');
-
-    if (numericPart) {
-      result += `${numericPart}: $${name},\n`;
-    } else if (nonNumericPart) {
-      result += `${nonNumericPart}: $${name},\n`;
-    }
+export type CssObjectType = { [key: string]: string | object };
+
+const specialCases: { [key: string]: string } = {
```

Comment:
What does `specialCases` stand for?

Reply:
Handles cases that are outside of the logical operation and represent an exception in the approach. Probably better to explain in person.